frontend/src/app/App.test.tsx

```diff
@@ -1169,33 +1160,6 @@ describe("App.handleNewSession", () => {
 
       expect(wrapper.find("DeployButton")).toHaveLength(0)
     })
-
-    it("button should be hidden for cloud environment", () => {
```

Comment:
Where did this test go?

Reply:
This test was moved to line 591 - title `Deploy button should be hidden for cloud environment`